auth-web/package-lock.json

```diff
@@ -8131,7 +8131,6 @@
       "version": "1.4.0",
       "inBundle": true,
       "license": "MIT",
-      "optional": true,
```

Comment:
Hongjing, what version of Node are you using?

(This is an inconsequential change, but it looks like your "npm i" works differently from whoever updated this file before you.)